### ISSUE TITLE
Accessibility: VoiceOver Can't Find Web Content If you Stop Interact

### ISSUE THREAD

**Issue by chigkim** [electron/electron#17207]
* **Electron Version:** 4.0.6
* **Operating System:** macOS 10.14.3

### Expected Behavior
If you stop interact with everything and go to the top level, VoiceOver should be able to find the main web content area and be able to interact with it again.

### Actual Behavior
VoiceOver only sees close, minimize, full screen buttons.

### To Reproduce
Just run the electron-quick-start example.

$ git clone https://github.com/electron/electron-quick-start

$ npm start

### Additional Information
Upon launching the app, VoiceOver will say Hello World Web Content.

From that point, you can immediately interact with the web content with vo(control+option)+shift+down, and then review the content with vo+left/right.

However, if you stop interact with the web content with vo+shift+up, you cannot get back into the web content area. VoiceOver only sees close, minimize, full screen buttons.

Only way is to restart the app.

**Comment by kloots** [comment]
Two of our customers at Slack have reported this problem. Here's the steps to reproduce with Slack:

1. Launch VoiceOver (press **command** + **f5**)
2. Launch Slack
3. Start interacting with Slack using VoiceOver navigation (e.g. press **control** + **option** + **shift** + 🔽 or **control** + **option** + ▶️  )
4. Stop interacting with Slack by pressing **control** + **option** + **shift** + 🔼 until you reach the top of the Window
5. Try to resume interacting with Slack by pressing **control** + **option** + **shift** + 🔽
6. Note how you cannot navigate back into Slack

**Comment by kloots** [comment]
From working with Apple on this:

> one possible cause is the `AtomNSWindow` only has three `AXChildren`: the close, fullscreen, and minimize buttons. In order for the contents to be accessible, there should be one or more other nodes in the Accessibility tree. I’d expect there to be AXGroups for each of the side bar and main content sections, with further descendent nodes nested in each.

![ax-inspector](https://user-images.githubusercontent.com/40818/73691990-4f690280-4688-11ea-965d-f4bdaae357a0.png)

**Comment by felixrieseberg** [comment]
Alright, good news and bed news! I just spent a bit of time trying to fix this, but as it turns out, it's shockingly simple: Voice Over needs to be running before the window is created. As long as that's the case, the web contents are in the accessibility tree and everything works as expected.

I've confirmed that this is now fixed in a recent version of Electron. I did not not go through the trouble of confirming which one, but everything is okay in Electron > 7.

I'm closing this issue, but please do let me know if there's anything I've missed.

**Comment by kloots** [comment]
@felixrieseberg I do think you missed something: this issue is still reproducible even if VoiceOver is running _before_ you start an Electron instance. [See the repro steps I provided above](https://github.com/electron/electron/issues/17207#issuecomment-581615553).

**Comment by kloots** [comment]
Worth noting this is still an issue and I believe the root cause is in Chromium. ([See Chromium issue #1111466](https://bugs.chromium.org/p/chromium/issues/detail?id=1111466))
